docs/readme.md

```diff
@@ -1,23 +1,42 @@
+**Legend :**
+1. Block directory
+2. Block editor content area
+3. Block options
 
 Using a system of Blocks to compose and format content, the new block-based editor is designed to create rich, flexible layouts for websites and digital products. Content is created in the unit of blocks instead of freeform text with inserted media, embeds and Shortcodes (there's a Shortcode block though).
 
 Blocks treat Paragraphs, Headings, Media, and Embeds all as components that, when strung together, make up the content stored in the WordPress database, replacing the traditional concept of freeform text with embedded media and shortcodes. The new editor is designed with progressive enhancement, meaning that it is back-compatible with all legacy content, and it also offers a process to try to convert and split a Classic block into equivalent blocks using client-side parsing. Finally, the blocks offer enhanced editing and format controls.
 
 The Editor offers rich new value to users with visual, drag-and-drop creation tools and powerful developer enhancements with modern vendor packages, reusable components, rich APIs and hooks to modify and extend the editor through Custom Blocks, Custom Block Styles and Plugins.
 
-## Quick Links
+## Quick links
 
--   Beginners: The [Create a Block Tutorial](/docs/designers-developers/developers/tutorials/create-block/readme.md) walks through creating a block plugin using the `@wordpress/create-block` package; a quick and easy way to start creating your own block.
+### [Create a Block Tutorial](/docs/designers-developers/developers/tutorials/create-block/readme.md)
+Learn how to create your first block for the WordPress block editor. From setting up your development environment, tools, and getting comfortable with the new development model, this tutorial covers all what you need to know to get started with the block editor.
 
--   [Gutenberg Architecture](/docs/architecture/readme.md) covers the conceptual and tool choices going into the Gutenberg project, from repository structure to packages and tools.
+- [Dynamic Blocks](https://developer.wordpress.org/block-editor/tutorials/block-tutorial/creating-dynamic-blocks/)
 
--   [Block API Reference](/docs/designers-developers/developers/block-api/README.md)
+### [Develop for the block editor](https://developer.wordpress.org/block-editor/developers/)
+Whether you want to extend the functionality of the block editor, or create a plugin based on it, you will find here all the information about the basic concepts you need to get started, the block editor APIs and its architecture.
 
--   [Contributors Guide](/docs/contributors/readme.md) covers how **you** can help; be it code, design, documentation, language, or triage, we need you to help make Gutenberg.
+- [Block Filters](https://developer.wordpress.org/block-editor/developers/filters/block-filters/)
+- [Editor Filters](https://developer.wordpress.org/block-editor/developers/filters/editor-filters/)
```

Comment:
I'd personally deemphasize filters entirely. I  don't think they should be seen as the primary way to extend the editor. I'd rather link to  "creating  blocks", "creating block patterns", "creating style variations", "thumbing the block editor".

Reply:
> I'd rather link to "creating blocks"

Do you mean here would be a better place for the "creating a block tutorial"?

Reply:
yes that's what I was suggesting, would love to hear other opinions though.;

Reply:
what if we switch the order and have the chapter of creating a block after the "develop for the block editor" one? Like i did here https://github.com/WordPress/gutenberg/pull/28142/commits/ebe929cea67feccc844bcbb6a9005e7e6053a2b1

I still think this is an important part and should have its own chapter.

Reply:
Oh actually, I think it needs to be first like it was. I got confused between this tutorial the "@wordpress/create-block" docs.

Reply:
Sorry for the back and forth and yeah once you switch back, this would look great for me as an iteration.